templates/api/update_email_confirm.html

```diff
@@ -8,6 +8,7 @@
                 <div class="card-body">
                     <h5 class="card-title">Update Email Request</h5>
                     <form method="POST">
+                        <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
```

Comment:
this not needed as:
 - api_endpoint.py is exempt from csrf anyhow
 - form is protected by unique code in url

Reply:
removed the csrf token line.